from mlflow.exceptions import MlflowException as MlflowException
from mlflow.protos.databricks_pb2 import BAD_REQUEST as BAD_REQUEST, INVALID_PARAMETER_VALUE as INVALID_PARAMETER_VALUE
from mlflow.tracking.artifact_utils import get_artifact_repository as get_artifact_repository

def download_artifacts(artifact_uri: str | None = None, run_id: str | None = None, artifact_path: str | None = None, dst_path: str | None = None) -> str: ...
def load_text(artifact_uri: str) -> str: ...
def load_dict(artifact_uri: str) -> dict: ...
def load_image(artifact_uri: str): ...
